docs/DeveloperGuide.md

```diff
@@ -325,6 +325,46 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
       Use case ends.
 
+**Use Case: Edit an internship entry.**
+
+**MSS**
+
+1.  User edits an internship entry.
+2.  InternBuddy updates that particular internship entry.
```

Comment:
Is it also better to say "and displays success message"?

Reply:
Ok noted, will add the "display success message" in.